### ISSUE TITLE
OpenMPI osu_get_acc_latency gets memory registration warnings

### ISSUE THREAD

**Issue by sungeunchoi** [ofi-cray/libfabric-cray#667]
The osu_get_acc_latency benchmark gets a large number of warnings regarding memory registration for sizes > 131072.

```
% srun -n2 -N2 -t00:2:00 --exclusive ./osu_get_acc_latency
# OSU MPI_Get_accumulate latency Test v5.0
# Window creation: MPI_Win_allocate
# Synchronization: MPI_Win_flush
# Size          Latency (us)
...
libfabric:gni:mr:__mr_cache_create_registration():1345<warn> [19617:1] failed to register memory with uGNI, ret=GNI_RC_INVALID_PARAM
libfabric:gni:mr:__mr_cache_search_inuse():1180<warn> [19617:1] failure in registration, can't really recover at this point since we've already retired the existing entries
```

**Comment by sungeunchoi** [comment]
Related to #277

**Comment by sungeunchoi** [comment]
Fixed by #809, though additional quieting will happen with #804.
